### PR TITLE
chore(leaving-ibm): set up codesandbox for leaving-ibm

### PR DESCRIPTION
**Pull request by annawen1**
### Related Ticket(s)

Web Components: Develop Leaving IBM component #3327

### Description

Set up codesandbox for Leaving IBM web component

### Changelog

**New**

- codesandbox example for leaving ibm web component

### CODE REVIEW

File: packages/web-components/examples/codesandbox/components/leaving-ibm/src/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal';
+import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal-body';
+import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal-heading';
+import 'carbon-web-components/es/components/modal/modal-header';
+import 'carbon-web-components/es/components/modal/modal-close-button';
+import 'carbon-web-components/es/components/button/button';
```
Comment (asudoh) [inline packages/web-components/examples/codesandbox/components/leaving-ibm/src/index.js:15]: Please refer to: #4437

```suggestion
import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal.js';
import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal-body.js';
import '@carbon/ibmdotcom-web-components/es/components/leaving-ibm/leaving-ibm-modal-heading.js';
import 'carbon-web-components/es/components/modal/modal-header.js';
import 'carbon-web-components/es/components/modal/modal-close-button';
import 'carbon-web-components/es/components/button/button.js';
```

**Review by asudoh — COMMENTED** [review]
Good update! In addition to `.js` extension thing, great if SnowPack (instead of Parcel) is used in future (using Parcel is fine for this PR, though) given:

1. SnowPack is faster, esp. in CodeSandbox
2.  Parcel `1.x` has maintenance concerns, though `2.x` hasn't been out

Thanks @annawen1!

File: packages/web-components/examples/codesandbox/components/leaving-ibm/.babelrc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          "not android > 0",
+          "not edge > 0",
+          "not ie > 0",
+          "not ie_mob > 0"
+        ]
+      }
+    ]
+  ],
+  "plugins": [["@babel/plugin-transform-runtime", { "version": "7.3.0" }]]
+}
```
Comment (asudoh) [inline packages/web-components/examples/codesandbox/components/leaving-ibm/.babelrc:22]: `.babelrc` is not needed with Snowpack.

**Review by asudoh — APPROVED** [review]
Thank you @annawen1 for the update! Just one thing.

**Review by kennylam — APPROVED** [review]
LGTM! Thanks @annawen1

File: packages/web-components/examples/codesandbox/components/leaving-ibm/.babelrc (hunk 2 above)
Comment (annawen1) [reply packages/web-components/examples/codesandbox/components/leaving-ibm/.babelrc]: done!

Comment (asudoh) [reply packages/web-components/examples/codesandbox/components/leaving-ibm/.babelrc]: Cool thanks @annawen1!

**Pull request merged**